pages/[walletId]/index.tsx

```
@@ -184,264 +562,573 @@ const Home: NextPage = () => {
+              </DropdownMenu>
+            </div>
+          )}
+          
+          {/* Add Member Form */}
+          {showAddMember && (
+            <FormPanel title="Add New Member" onClose={() => { setShowAddMember(false); setNewMemberWallet(''); setNewMemberShares(0); setNewMemberType('wallet') }} className="mb-6">
+              
+              {/* Member Type Selector - only show for NFT model fanouts */}
+              {fanoutData.data?.fanout?.membershipModel === 1 && (
+                <div className="mb-4">
+                  <label className="block text-sm font-medium text-gray-400 mb-2">Member Type</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)